purchase_packaging_level_qty/models/purchase_order.py

```diff
@@ -26,9 +32,20 @@ def _compute_purchase_transport_packaging_level(self):
     def _compute_total_transport_packaging_qty(self):
         for order in self:
             if order.transport_packaging_level_id:
-                total_transport_qty = sum(order.mapped("order_line.transport_qty"))
-                order.total_transport_qty = "{} {}".format(
-                    total_transport_qty,
+                order.total_transport_qty = sum(
+                    order.mapped("order_line.transport_qty")
```

Comment:
Could we take the chance to improve this?
We should use `read_group` + `sum` operator instead.

Reply:
It is more performant ?
Updating code...

Reply:
>  We should use read_group + sum operator instead.

Of course. For 2 reasons: just one read + the sum is delegated to the db.

Reply:
`transport_qty` is not a stored field that can be used to sql sum